app/src/main/java/com/android/streetworkapp/ui/progress/Progression.kt

```diff
@@ -165,12 +173,20 @@ fun ProgressScreen(
               }
               DashboardStateProgression.Training -> {
 
-                enumValues<ExerciseAchievement>().forEach {
-                  it.achievement.description =
-                      "Record : 0.0 sec" // TODO: use the progression MVVM with current record
-                  Box(modifier = Modifier.testTag("exerciseItem${it.name}")) {
-                    AchievementItem(it.achievement, true)
-                    HorizontalDivider(thickness = 0.5.dp, color = Color.LightGray)
+                currentWorkout?.workoutSessions?.forEach { workout ->
+                  val date = workout.startTime.toFormattedString()
+                  workout.exercises.forEach {
+                    val newAchievement =
+                        Achievement(
+                            exerciseNameToIcon(it.name),
+                            "You trained your ${it.name} !",
```

Comment:
Avoid using hard-coded strings and use resource strings instead.

Reply:
Done, thank you!